src/main/java/org/datadog/jenkins/plugins/datadog/listeners/DatadogSaveableListener.java

```diff
@@ -0,0 +1,54 @@
+package org.datadog.jenkins.plugins.datadog.listeners;
+
+import hudson.Extension;
+import hudson.XmlFile;
+import hudson.model.Saveable;
+import hudson.model.listeners.SaveableListener;
+import net.sf.json.JSONArray;
+import org.datadog.jenkins.plugins.datadog.DatadogClient;
+import org.datadog.jenkins.plugins.datadog.DatadogEvent;
+import org.datadog.jenkins.plugins.datadog.DatadogUtilities;
+import org.datadog.jenkins.plugins.datadog.events.ConfigChangedEventImpl;
+
+import java.util.Map;
+import java.util.Set;
+import java.util.logging.Logger;
+
+/**
+ * This class registers an {@link SaveableListener} to trigger events and calculate metrics:
+ * - When an saveable gets changed, the {@link #onChange(Saveable, XmlFile)} method will be invoked.
+ */
+@Extension
+public class DatadogSaveableListener  extends SaveableListener {
```

Comment:
What's this annotation ? Why other Listeners don't have it ?

Reply:
Good catch i missed them. It registers the listener in Jenkins.